app/build.gradle

```diff
@@ -111,11 +116,16 @@ dependencies {
     implementation 'com.android.support:appcompat-v7:27.1.1'
     implementation 'com.android.support:recyclerview-v7:27.1.1'
     implementation 'com.google.code.gson:gson:2.8.2'
-    implementation 'io.left.rightmesh:rightmesh-library:0.5.0'
+
+    implementation ('io.left.rightmesh:rightmesh-library:0.9.0'){
+        changing = true
+    }
     implementation fileTree(include: ['*.jar'], dir: 'libs')
 
     androidTestImplementation 'com.android.support:support-annotations:27.1.1'
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'android.arch.persistence.room:testing:1.1.0'
     implementation 'com.airbnb.android:lottie:2.5.0-rc1'
+
+    implementation 'com.vincent.filepicker:MultiTypeFilePicker:1.0.7'
```

Comment:
Do we know the licensing of this plugin and/or what kind of size it adds to the APK?

Reply:
it uses Apache 2.0